.github/workflows/main_ci.yml

```diff
@@ -81,8 +81,9 @@ jobs:
       matrix:
         build_type: [Debug, Release]
         sharedlibs: [OFF, ON]
+        std: [14, 17, 20]
 
-    runs-on: windows-2019
+    runs-on: windows-latest
```

Comment:
Please hardcode the specific windows version you want; GitHub updates it occasionally which changes the generator.